style.css

```diff
@@ -0,0 +1,342 @@
+  padding-bottom: 20px;
+}
+
+.search-icon {
+  display: flex;
+  position: absolute;
+  left: 148px;
+  align-items: center;
+  justify-content: center;
+  width: 14px;
+  height: 14px;
+}
+
+.header__user-block {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  text-transform: uppercase;
+  font-weight: bold;
+}
+
+.header__cart {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.header__cart_counter {
+  height: 14px;
+  width: 14px;
+  border-radius: 50%;
+  background-color: #000;
+  color: #fff;
+  margin-right: -20px;
+  margin-left: 5px;
+}
+
+.header__language-block {
+  display: flex;
+  align-items: center;
+  justify-content: flex-end;
+  margin: 0;
+  padding: 0;
+}
+
+.header__language-image {
+  width: 20px;
+  height: 20px;
+}
+
+/* top-content */
+
+.top-content {
+  background-image: url(img/header-img.png);
+  display: block;
+  /* display: flex;
+  justify-content: flex-start; */
+  width: 100%;
+  height: 253px;
+  margin-bottom: 67px;
+}
+
+.top-content__block {
+  display: flex;
+  flex-direction: column;
+  width: 209px;
+  padding-left: 54px;
+  padding-top: 78px;
+  justify-content: center;
+  align-items: flex-start;
+}
+
+.top-content__h1 {
+  font-size: 30px;
+  font-weight: bold;
+  line-height: 29px;
+  text-transform: uppercase;
+  color: #ffffff;
+  text-align: left;
+  padding-bottom: 6px;
+}
+
+.link__white {
+  color: #ffffff;
+  text-align: left;
+}
+
+.arrow {
+  padding-left: 7px;
+}
+
+/* catalog */
+
+.catalog {
+  margin-bottom: 52px;
+}
+
+.catalog__title {
+  font-size: 22px;
+  font-weight: bold;
+  line-height: 21px;
+  text-align: center;
+  text-transform: uppercase;
+  margin-bottom: 15px;
+  font-weight: bold;
+}
+
+.catalog__items {
+  display: flex;
+  justify-content: space-between;
+}
+
+.catalog__item {
+  display: block;
+  position: relative;
+  width: 208px;
+  height: 261px;
+}
+
+.catalog__item:not(:last-child):after {
+  content: "";
+  opacity: .2;
+  background: #000000;
+  height: 101px;
+  position: absolute;
+  top: 36px;
+  right: -15px;
+  width: 1px;
+}
+
+.catalog-item__image {
+  width: 170px;
+  height: 173px;
+  margin-bottom: 24px;
+  padding-left: 21px;
+}
+
+.catalog-item__title {
+  font-size: 12px;
+  line-height: 11px;
+  text-align: left;
+  margin-bottom: 10px;
+  padding-left: 11px;
+}
+
+.catalog-item__description {
+  display: grid;
+  grid-template-columns: 2fr 1fr;
+  padding-left: 11px;
+}
+
+.catalog-item__price {
+  display: flex;
+  align-items: center;
+  justify-content: flex-start;
+  font-size: 14px;
+  font-weight: bold;
+  line-height: 13px;
+  text-align: left;
+}
+
+.catalog-item__varities {
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+}
+
+.catalog-item__grey {
+  height: 7px;
+  width: 7px;
+  border-radius: 50%;
+  background-color: #C4C4C4;
+}
+
+.catalog-item__black {
+  height: 7px;
+  width: 7px;
+  border-radius: 50%;
+  background-color: #0E1211;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected "#0E1211" to be "#0e1211" (color-hex-case)](https://app.codacy.com/gh/WebHeroSchool/PolinaYartseva_axion/pullRequest?prid=5569682)